acceptance-tests/tests/src/test/resources/jsonrpc/eth/getBlockBy/test-cases/04_update_forkchoice.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
This test also starts with 04. I expected the prefixes would be unique like the other rpc tests

Reply:
good catch, I missed that after inserting it...looks like I need to update the test names afterall :)